torch/utils/cpp_extension.py

```diff
@@ -1144,12 +1155,13 @@ def CUDAExtension(name, sources, *args, **kwargs):
     return setuptools.Extension(name, sources, *args, **kwargs)
 
 
-def include_paths(cuda: bool = False) -> List[str]:
+def include_paths(cuda: bool = False, xpu: Optional[bool] = None) -> List[str]:
```

Comment:
could you unify this function to accept a backend string (like `cuda` or `xpu`) instead of a bool value for each backend.